### ISSUE TITLE
Example command 

### ISSUE THREAD

**Issue by yoonchoi-neuro** [Center-of-Imaging-Biomarker-Development/chp_seg#1]
Dear github master, 

 First of all, thank you for sharing such a useful tool. 
 I followed all the instructions to apply the tool to images, but it is a bit hard to use the command for people not familiar with docker.  
 Can you please give any specific command example? 

 Thank you.

**Comment by hettk** [comment]
Dear Yoonchoi-neuro,

Thank you for your interest in testing our tool and providing us some valuable feedback, we will work on providing a more descriptive guideline to use the software with Docker.

I will use your last message to give some examples. The previous missing element was the 

sudo docker run -v /hdd2/ABIDE/image_data_T1/:**/data/in** -v /hdd2/ABIDE/test/:**/data/out** kilianhett/chp_seg:1.0.0 --sequence_type T1 --name_pattern ABIDE

- **/hdd2/ABIDE/image_data_T1** has to be a folder that contains all images (_the software will not look into subfolders_)
- **/hdd2/ABIDE/test/** is the target folder where all the output will be stored. 
- **--sequence_type** The current version of our software has been trained separately using **T1,** **T2,** or **FLAIR** sequence. As you want to run the segmentation software on T1-weighted images, the option you put is correct. 

- **--name_pattern** This will filter out scans based on their filenames, playing as follows for the option "ABIDE" = *ABIDE* (please do not add * as the regular expression is specified in the code). Also, note that this option is optional, without this option the software will be run on all files in the folder **/hdd2/ABIDE/image_data_T1** if the folder contains other types of files (such as .json), I recommend you use ".nii".


Thank you,
Kilian

**Comment by tobias-kaufmann** [comment]
Dear Kilian
Thanks for the helpful descriptions. 
Could you elaborate also on the text output of the software? The last line returns “Total volume = xxxx voxels”. When I compare this to the output of 'fslstats -V' (number of non-zero voxels in mask / volume of mask in mm3) I see that the values do not align. So what is the last line of the text output referring to?
Thanks!

**Comment by CenterImagingBiomarkerDevelopment** [comment]
Hello Tobias,

The output was a remaining code to monitor the segmentation. It has no value as it corresponds to the number of voxels of the mask before back-transformation to the native input scan space.

Also, I wanted to let you know that we just released a software upgrade version. We corrected this confusing point and extended the estimation of the volume for the right and left ChP in addition to the total volume. Finally, the software now generates automatically volumetric reports in CSV format.

Best,
Kilian
